back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -33,6 +33,24 @@ def reorder
     end
   end
 
+  def list_areas_of_projects
```

Comment:
I'm not entirely convinced this file is the best location for this action (since it is returning a list of `areas`, not `admin_publications`). In the end, I chose to put it here because it is so intimately tied to `admin_publications` in how it finds the unique areas for visible (no-draft) `projects`.  
  
Maybe I should move this to the `areas_controller`?